PasteIntoFile/Properties/Resources.resx

```diff
@@ -277,6 +277,13 @@ These setting will take effect on the next login.</value>
   <data name="str_clear_clipboard_tooltip" xml:space="preserve">
     <value>Clears the clipboard after its content has been saved.</value>
   </data>
+  <data name="str_disable_live_clipboard_update" xml:space="preserve">
+    <value>Disable live clipboard update</value>
+    <comment>Checkbox to disable live clipboard updates in the preview/rename dialog</comment>
```

Comment:
Instead of enabling a checkbox to disable a feature, can we avoid the inverse logic?